src/yunohost/app.py

```diff
@@ -1254,11 +1253,11 @@ def _fetch_app_from_git(app):
                 branch = app[tree_index+6:]
             try:
                 subprocess.check_call([
-                    'git', 'clone', '--depth=1', url, app_tmp_folder])
+                    'git', 'clone', '--depth=1', url, extracted_app_folder])
```

Comment:
It's normally declared here https://github.com/YunoHost/yunohost/pull/163/files#diff-c98834a0df97adaa917878824a06b38fR1247